RATapi/controls.py

```diff
@@ -39,40 +40,98 @@
 class Controls(BaseModel, validate_assignment=True, extra="forbid"):
     """The full set of controls parameters for all five procedures that are required for the compiled RAT code."""
 
+    model_config = ConfigDict(use_attribute_docstrings=True)
```

Comment:
I think it'd be better to include this as a keyword argument of the model, in line with the other config options.